main.go

```diff
@@ -268,6 +269,9 @@ func main() { //nolint:funlen,maintidx,gocyclo
 			&rbacv1.RoleBinding{}: {
 				Namespaces: oDHCache,
 			},
+			&ofapiv1alpha1.Subscription{}: {
```

Comment:
this is likely unrelated

Reply:
forgot to push the new change which reverted this part.....